src/index.html

```diff
@@ -11,7 +11,212 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="#" method="post">
+      <fieldset class="group group-pesonal">
+        <legend>Personal information</legend>
+        <label
+          class="label"
+          for="surname"
+        >
+          Surname:
+          <input
+            class="input"
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label
+          class="label"
+          for="name"
+        >
+            Name:
+          <input
+            class="input"
```

Comment:
Each level of nesting requires **exactly** 2 indents
```suggestion
        <label
          class="label"
          for="name"
        >
          Name:
          <input
            class="input"
```